doc/_static/custom_styles.css

```diff
@@ -160,3 +160,35 @@ div#driver-popover-item .tour-progress-indicator {
 a[href="#join-our-community"] + ul li.toc-h3 {
   display: none;
 }
+
+/*Version selector and header*/
+/* #versions-header {
+  margin: 0 10px;
+}
+
+#select-versions, #select-versions option {
+  width: 90%;
+}
+
+#select-versions {
+  margin: 10px;
+} */
+
+#versions-header {
+  color: rgba(0,0,0,.65);
+  font-size: 0.8rem;
+  text-transform: uppercase;
+  margin-top: 1.1rem;
+}
+
+#select-versions {
+  margin-bottom: 7px;
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  font-size: 90%;
+  color: rgba(0,0,0,.65);
+  padding: 2px;
```

Comment:
```suggestion
  padding: 2px;
  background-color: transparent;
```

An ugly gray background was shown in the dropdown.

### Before

![Selección_066](https://user-images.githubusercontent.com/365293/91383844-c1569400-e7f2-11ea-9f64-b1a6916d0fb0.png)

### After

![Selección_067](https://user-images.githubusercontent.com/365293/91383853-c61b4800-e7f2-11ea-9c2e-64b510bbe481.png)